### DISCUSSION TITLE
Self host a shadcn registry

### DISCUSSION THREAD

**Discussion by skylineagle** [shadcn-ui/ui#5870] [Q&A]
Hey, i would really want to work with the shadcn/ui project, but my work is in an airgapped enviroment without access to public internet, we have are own npm mirror and so on, how can i deploy a mirror of shadcn/ui and use it without internet access?

tanks :)

**Comment by zaaakher** [comment]
Checkout this new section in the shadcn-ui docs: https://ui.shadcn.com/docs/registry
